src/lib.rs

```diff
@@ -106,7 +124,7 @@ pub trait Attestation {
     
             return Ok(())
         } else {
-            return sc_error!("there is not registered user under key");
+            return sc_error!("impossible error");
```

Comment:
You don't need the else. Maybe early exit? Also, why is this an "impossible error"?

Reply:
it is needed for rust - otherwise it won't compile. It is impossible as it is ensured that there is Data under the given key.